style.css

```diff
@@ -53,45 +49,71 @@ ul {
   background: #f4f1f0;
 }
 
-/* Nav Bar */
 header {
-  font-size: 18px;
-  display: flex;
-  align-items: center;
-  justify-content: space-between;
   height: 48px;
-  text-decoration: none;
-  color: #fff;
   background: #3c3a39;
+  padding: 0 12px;
+  overflow: hidden;
 }
 
-nav {
-  padding-right: 32px;
+@media only screen and (min-width: 992px) {
+  header {
+    padding: 0 100px;
+  }
 }
 
-.fa-bars {
-  color: white;
-}
-
-#logo {
-  font-family: "Roboto", sans-serif;
-  font-size: 18px;
-  font-weight: 700;
+.topnav a {
+  font-family: 'Inter', sans-serif;
   font-style: normal;
-  line-height: 24px;
-  display: flex;
-  align-items: center;
-  padding: 10px 12px;
-  text-align: center;
   text-decoration: none;
+  font-size: 15px;
+  line-height: 20px;
   color: #fff;
+  display: block;
+  float: right;
+  padding: 14px 16px;
+}
+
+.topnav a:hover {
+  background-color: #ddd;
+  color: black;
+}
+
+.topnav a.logo {
+  font-size: 20px;
+  font-weight: 700;
+  display: block;
+  float: left;
+}
+
+.topnav a:not(:first-child) {
+  display: none;
+}
+
+.topnav a.bars {
+  float: right;
+  display: block;
+}
+
+@media only screen and (min-width: 992px) {
```

Comment:
- [ ] I think a screen with 992 pixels is too wide for a Mobile screen size view. As previously explained in an early comment. it would be useful to use a recommended screen size of 768px instead of 992px